[ACA] [371940] Add subPath documentation. (#125940)
Azure/azure-content: articles/container-apps/storage-mounts.md

1. In **Mount path**, enter the absolute path in the container to mount the volume.

1. In **Sub path (optional)**, enter the path in the volume to mount. If not specified, the volume root is mounted. For more information see (#sub-path).

1. Select **Save** to save changes and exit the context pane.

1. Select **Create** to create the new revision.

::: zone-end

### Sub path

When mounting a file share from Azure Files, in addition to the mount path, you can also specify a sub path.

- The mount path is the path in the container where you want to mount the volume.

> - The sub path is the path in the volume you want to mount.

Reviewer: ```suggestion
- **Mount path**: The path in the container where you want to mount the volume.
- **Sub path**: The path in the volume you want to mount.
```

The sub path is optional. If not specified, the volume root is mounted.

> The sub path should be a relative path from the volume root. The sub path should not start with `/`. Specifying a sub path that starts with `/` might prevent your container app from starting up. For example, `my-volume-folder` is a valid sub path, whereas `/my-volume-folder` is not.

Reviewer: ```suggestion
The sub path is a relative path from the volume root. The sub path should not start with `/`. Specifying a sub path that starts with `/` might prevent your container app from starting up. For example, `my-volume-folder` is a valid sub path, where `/my-volume-folder` is not.
```

The sub path can refer to either a folder or a file in the volume.
- If the sub path refers to a folder, the mount path should refer to an empty folder in the container.
- If the sub path refers to a file, the mount path should refer to a file that does not already exist in the container. For example, if the sub path is `my-volume-folder/my-volume-file` and the mount path is `/my-container-folder/my-container-file`, the folder `/my-container-folder` should exist in the container, but should not already contain the file `my-container-file`.

If the sub path has a trailing `/`, whether it refers to a folder or file, the trailing `/` is ignored.

> For more information see [using subPath](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath).

Reviewer: Add "Related content"

1. In **Mount path**, enter the absolute path in the container to mount the volume.

1. In **Sub path (optional)**, enter the path in the volume to mount. If not specified, the volume root is mounted. For more information see (#sub-path).

1. Select **Save** to save changes and exit the context pane.

1. Select **Create** to create the new revision.

::: zone-end

### Sub path

When mounting a file share from Azure Files, in addition to the mount path, you can also specify a sub path.

- **Mount path**: The path in the container where you want to mount the volume.
- **Sub path**: The path in the volume you want to mount.

The sub path is optional. If not specified, the volume root is mounted.

The sub path is a relative path from the volume root. The sub path should not start with `/`. Specifying a sub path that starts with `/` might prevent your container app from starting up. For example, `my-volume-folder` is a valid sub path, where `/my-volume-folder` is not.

> The sub path can refer to either a folder or a file in the volume.
> - If the sub path refers to a folder, the mount path should refer to an empty folder in the container.
> - If the sub path refers to a file, the mount path should refer to a file that does not already exist in the container. For example, suppose the sub path is `my-volume-folder/my-volume-file` and the mount path is `/my-container-folder/my-container-file`. The folder `/my-container-folder` should exist in the container, but it should not already contain the file `my-container-file`.

Reviewer: ```suggestion
The sub path can refer to either a folder or a file in the volume.

- If the sub path refers to a folder, the mount path should refer to an empty folder in the container.

- If the sub path refers to a file, the mount path should refer to a file that does not already exist in the container.

    For example, suppose the sub path is `my-volume-folder/my-volume-file.txt`, and the mount path is `/my-container-folder/my-container-file`. The folder `/my-container-folder` should already exist in the container but should not yet contain the file `my-container-file.txt`.
```

> If the sub path has a trailing `/`, whether it refers to a folder or file, the trailing `/` is ignored.

Reviewer: How would a trailing slash relate to a file?

> If the sub path has a trailing `/`, whether it refers to a folder or file, the trailing `/` is ignored.

Reviewer: ```suggestion
If the sub path trailing slashes are ignored.
```

::: zone-end

### Sub path

When mounting a file share from Azure Files, in addition to the mount path, you can also specify a sub path.

- **Mount path**: The path in the container where you want to mount the volume.
- **Sub path**: The path in the volume you want to mount.

The sub path is optional. If not specified, the volume root is mounted.

The sub path is a relative path from the volume root. The sub path should not start with `/`. Specifying a sub path that starts with `/` might prevent your container app from starting up. For example, `my-volume-folder` is a valid sub path, where `/my-volume-folder` is not.

The sub path can refer to either a folder or a file in the volume.

- If the sub path refers to a folder, the mount path should refer to an empty folder in the container.

- If the sub path refers to a file, the mount path should refer to a file that does not already exist in the container.

    For example, suppose the sub path is `my-volume-folder/my-volume-file.txt`, and the mount path is `/my-container-folder/my-container-file`. The folder `/my-container-folder` should already exist in the container but should not yet contain the file `my-container-file.txt`.

> If the sub path trailing slashes are ignored.

Reviewer: ```suggestion
Any sub path trailing slashes are ignored.
```
